src/generate_ds.py

```diff
@@ -87,16 +87,16 @@ def generate_ds(context: MLClientCtx, input_ds: str):
     df.drop(inplace=True, columns=["answer", "explanation"])
     context.log_dataset("new-train-ds", df)
     context.logger.info("Dataframe logged")
-
-    # Upload the dataset to HuggingFace
-    hf_dataset = Dataset.from_pandas(df)
-    login(token=hf_token)
-
-    # Create a new repository on the Hugging Face Hub
-    repo_id = "mlrun/banking-orpo-new"
-    create_repo(repo_id, repo_type="dataset", exist_ok=True)
-
-    # Push the dataset to the Hub
-    hf_dataset.push_to_hub(repo_id)
-    context.log_result("dataset", repo_id)
-    context.logger.info("Dataset uploaded to HF")
+    if log_to_hf:
```

Comment:
It is better to change it to `hf_repo_id` if the user has their own repo and wishes to use it instead of pushing only to mlrun.

Reply:
Done